### PR TITLE
Backports Adherent

### PR DESCRIPTION
**Pull request by quardbreak**
Opening for diff. 
Todo: add multiple hands.

## Changelog
:cl:
rscadd: Backported Adherent species from Baystation12.
/:cl:

### CODE REVIEW

File: mods/species/adherent/clothing/head.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/obj/item/clothing/head/Initialize()
+	. = ..()
+	LAZYDISTINCTADD(bodytype_restricted, BODYTYPE_ADHERENT)
```
Comment (MistakeNot4892) [inline mods/species/adherent/clothing/head.dm]: Needs a check on "exclude".

File: mods/species/adherent/datum/species.dm [hunk 2]
```diff
@@ -0,0 +1,218 @@
+/decl/species/adherent
+	name = SPECIES_ADHERENT
+	name_plural = "Adherents"
+
+	description = "The Vigil is a relatively loose association of machine-servitors, Adherents, \
```
Comment (MistakeNot4892) [inline mods/species/adherent/datum/species.dm]: I feel like this needs to be trimmed like other species, but I'm not sure how you could reduce it without making it unclear what the adherent are.

**Review by MistakeNot4892 — DISMISSED** [review]
I want to go over all the species lore when we stop backporting them and make it clean/consistent so imo this is fine.

**Comment by comma** [comment]
Conmblicts

**Review by MistakeNot4892 — APPROVED** [review]

**Pull request merged**